AirCasting/CoreData/MeasurementStreamStorage.swift

```diff
@@ -195,8 +195,10 @@ final class HiddenCoreDataMeasurementStreamStorage: MeasurementStreamStorageCont
         return session
     }
 
-    func updateMeasurements(stream: MeasurementStreamEntity, newMeasurements: NSOrderedSet) throws {
-        stream.measurements = newMeasurements
+    func removeAllMeasurements(in stream: MeasurementStreamEntity, except measurementsToLeave: [MeasurementEntity]) {
```

Comment:
Great, thank you for changing that! This function name and logic was very misleading